analyzer-core/src/main/java/ffb/analyzer/core/queue/PriorityTask.java

```diff
@@ -0,0 +1,32 @@
+package ffb.analyzer.core.queue;
+
+
+
+public class Task implements Comparable<Task> {
```

Comment:
How about `PrioritizedTask` that inherits from a base `Task` interface? We could use that same interface for non-prioritized tasks